### PR TITLE
expand the examples section

### PR DESCRIPTION
**Pull request by severo**

### CODE REVIEW

File: docs/hub/datasets-viewer-embed.md [hunk 1]
```diff
@@ -58,15 +58,56 @@ It will open a modal with the iframe code that you can copy and paste into your
...
-- [Metaflow + Outerbounds](https://www.linkedin.com/posts/eddie-mattia_the-team-at-hugging-facerecently-released-activity-7219416449084272641-swIu)
-- [AutoTrain](https://x.com/abhi1thakur/status/1813892464144798171)
+Open a [pull request](https://github.com/huggingface/hub-docs/blob/main/docs/hub/datasets-viewer-embed.md) if you want to appear in this section!
 
-Dataset creators also use the embedded viewer to showcase their datasets on their websites, spaces or apps:
+### Tool: ZenML
 
-- [librarian-bots/alpaca-style-datasets gallery](https://huggingface.co/spaces/librarian-bots/Alpaca-Style-Datasets-explorer)
-- [Electric Vehicle Charge Finder app](https://x.com/calebfahlgren/status/1813356638239125735)
+[`htahir1`](https://huggingface.co/htahir1) shares code in a [blog post](https://www.zenml.io/blog/embedding-huggingface-datasets-visualizations-with-zenml) to visualize a Hugging Face dataset within a [ZenML](https://huggingface.co/zenml) pipeline.
```
Comment (davanstrien) [inline docs/hub/datasets-viewer-embed.md:67]:
```suggestion
[`htahir1`](https://huggingface.co/htahir1) shares a [blog post](https://www.zenml.io/blog/embedding-huggingface-datasets-visualizations-with-zenml) showing how you can use the [ZenML](https://huggingface.co/zenml) integration with the Datasets Viewer to visualize a Hugging Face dataset within a  ZenML pipeline.
```

File: docs/hub/datasets-viewer-embed.md [hunk 2]
```diff
@@ -58,15 +58,56 @@ It will open a modal with the iframe code that you can copy and paste into your
...
+</a>
+</div>
+
+### App: Electric Vehicle Charge Finder
+
+[`cfahlgren1`](https://huggingface.co/cfahlgren1) [embeds](https://x.com/calebfahlgren/status/1813356638239125735) the dataset viewer in the [Electric Vehicle Charge Finder app](https://charge-finder.vercel.app/).
+
+<div class="flex justify-center">
+<a href="https://charge-finder.vercel.app">
+<img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/dataset-viewer-embed-example-gallery.map"/>
```
Comment (davanstrien) [inline docs/hub/datasets-viewer-embed.md:111]:
```suggestion
<img src="https://huggingface.co/datasets/huggingface/documentation-images/resolve/main/hub/dataset-viewer-embed-example-map.png"/>
```

Comment (davanstrien) [reply docs/hub/datasets-viewer-embed.md:111]: think it should be this link?

**Review by davanstrien — COMMENTED** [review]
small suggestion and think the final image link needs updating?

**Review by davanstrien — APPROVED** [review]
think the final image link still needs updating but looks good otherwise

**Comment by severo** [comment]
thanks!

**Pull request merged**